code.rb

```diff
@@ -0,0 +1,143 @@
+# MASTERMIND #
+class Mastermind
+  #set up the board
+
+  def initialize
+    @turn = 1
+  #set up the players
+    @board = Board.new
```

Comment:
I get an error here, because your Board class requires an argument to initialize.